src/services/apimap-field-builder.js

```diff
@@ -6,7 +6,13 @@ function ApimapFieldBuilder(model, column, options) {
 
   function isRequired() {
     // eslint-disable-next-line
-    return column._autoGenerated !== true && column.allowNull === false;
+    return column._autoGenerated !== true 
+      && (column.allowNull === false || column.primaryKey);
```

Comment:
This change looks to be another dedicated fix.
If we merge it like this, there will be no traces of such change in the generated changelog.

Reply:
Ok so according to your response in the tests comment,
if `autoIncrement` is set to `true`, `_autoGenerated` internal value is defined as `true`.
No problem here 👌 

However, I am still convinced, this is change that is not related to the `isGenerated` topic (and thus deserves a dedicated contribution (without it you won't have any trace of this change in the changelog).